.github/workflows/actions.yml

```diff
@@ -18,8 +18,8 @@ jobs:
   qemu-user-static:
     runs-on: ubuntu-latest
     env:
-      VERSION: 6.1.0-8
-      ORIGIN_VERSION: 6.1+dfsg-8
+      VERSION: 6.2.0-3
+      ORIGIN_VERSION: 6.2+dfsg-3
```

Comment:
Binary file for this version is not available in the pubic repo http://ftp.de.debian.org/debian/pool/main/q/qemu/, qemu-user-static_*

It can be update to 7.0+dfsg-7(http://ftp.de.debian.org/debian/pool/main/q/qemu/qemu-user-static_7.0+dfsg-7_amd64.deb

Reply:
Fixed in c68143860abc313180e76591da647b33f5c39f16